CameraXTfLite/build.gradle

```diff
@@ -37,6 +37,7 @@ buildscript {
 allprojects {
     repositories {
         google()
+        mavenCentral()
         jcenter()
```

Comment:
Sorry for the review comment that's unrelated to insets!

Was the goal here to get rid of `jcenter()`? It looks like it's the tensorflow dependencies that are blocking this removal.

Reply:
yeah, jcenter() is deprecated(will address this in all of other samples). for this one I will work with TFL team to find a fix in the future changes. thank you!